cluster/examples/kubernetes/ceph/storageclass-bucket-retain-external.yaml

```diff
@@ -1,18 +0,0 @@
-apiVersion: storage.k8s.io/v1
```

Comment:
I don't see this example listed in ceph-examples.md. Are there any other missing examples related to external clusters that should be added to that page?

Reply:
No, this one was missed in the initial PR.